test/test_events.py

```diff
@@ -16,6 +16,20 @@ def test_receiving_events():
     eq(event[1], 'py!')
     eq(event[2], [vim.current.buffer.number])
 
+@with_setup(setup=cleanup)
+def test_sending_notify():
+    # notify after notify
+    vim.session.notify('vim_command', "let g:test = 3")
+    cmd = 'call rpcnotify(%d, "test-event", g:test)' % vim.channel_id
```

Comment:
@Shougo `%` is not removed. Python developers were considering deprecating it and removing some time after, but this decision did not pass through. You may see now that https://docs.python.org/3/library/stdtypes.html#old-string-formatting is talking about string formatting method which has some certain issues (first note), but not about deprecated formatting method.